### PR TITLE
Second run to improve java 11

### PR DESCRIPTION
**Pull request by farodin91**
I disabled solr 7 for java 11. 

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [ ] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### CODE REVIEW

**Comment by farodin91** [comment]
@FlorianHockmann I've updated the PR.

File: .github/workflows/ci-backend-cql.yml [hunk 1]
```diff
@@ -98,18 +98,18 @@ jobs:
             java: 8
           - module: cql
             install-args: "-Pjava-11"
```
Comment (FlorianHockmann) [inline .github/workflows/ci-backend-cql.yml:100]: This file contains 12 test entries for Java 8, but only 3 for Java 11. Shouldn't all tests be also executed with Java 11?

File: docs/changelog.md [hunk 2]
```diff
@@ -149,6 +149,11 @@ we clean up all configurations. This could lead to unexpected new log lines. Ple
 !!! note 
     Log4j is only used for standalone server deployments and JanusGraph testing.
 
+##### Add support for Java 11
+
+All tests are no running on Java 11. Java 11 won't yet anymore the full build distribution.
```
Comment (FlorianHockmann) [inline docs/changelog.md:154]: (nitpick) Should be _are no**w** running on Java 11_.

But I don't understand the second sentence at all. The full build distribution is removed now?

File: janusgraph-server/pom.xml [hunk 3]
```diff
@@ -88,4 +88,12 @@
             </exclusions>
         </dependency>
     </dependencies>
+    <build>
+        <resources>
+            <resource>
+                <directory>${basedir}/src/main/resources</directory>
```
Comment (FlorianHockmann) [inline janusgraph-server/pom.xml:94]: I don't quite understand the purpose of this addition. I don't see any resources here under `src/main/resources`.

File: docs/changelog.md [hunk 4]
```diff
@@ -149,6 +149,14 @@ we clean up all configurations. This could lead to unexpected new log lines. Ple
 !!! note 
     Log4j is only used for standalone server deployments and JanusGraph testing.
 
+##### Add support for Java 11
+
+All tests are now running on Java 11. We encourage everyone to test out Java 11.
+
+!!! note
+    Currently, we aren't able to release full distribution build with Java 11.
```
Comment (FlorianHockmann) [inline docs/changelog.md:157]: I'm not sure what this means to be honest. We still build both distribution zip archives with Java 8, right? Does the full distribution then also require Java 8 as its runtime?
Or is this just informing users that we currently cannot build the full distribution with Java 11, although we also don't build the normal distribution with Java 8?

Comment (farodin91) [reply docs/changelog.md]: I added an extra line

**Comment by farodin91** [comment]
@FlorianHockmann  Would you like to check again?

File: docs/changelog.md (hunk 4 above)
Comment (FlorianHockmann) [reply docs/changelog.md]: Where? I don't see anything added in the diff, but the note is now missing completely.

**Comment by FlorianHockmann** [comment]
@farodin91 My problem with this PR is mostly that it's not clear to me what the state of support for Java 11 and 8 are after we merge this.

Are we supporting Java 11 in general, but use Java 8 by default, e.g., to build the distribution archives? Or are we already using Java 11 for everything?
And what about the runtime for users? Should users stay on Java 8 for now if they want highest stability and only adopt Java 11 on their own risk? Or should everybody now update to Java 11 as soon as possible?

The answers to these questions should be clear to users after they've read the changelog. I could maybe suggest a wording for the changelog, but I would need to know the answers myself first 😅

**Review by FlorianHockmann — APPROVED** [review]
We talked about this offline and changed the formulation for the changelog.

**Pull request merged**